configs/tasks/vln_r2r.yaml

```diff
@@ -0,0 +1,28 @@
+ENVIRONMENT:
+  MAX_EPISODE_STEPS: 500
+SIMULATOR:
+  FORWARD_STEP_SIZE: 0.25
+  TURN_ANGLE: 15
+  HABITAT_SIM_V0:
+    GPU_DEVICE_ID: 0
+  RGB_SENSOR:
+    WIDTH: 512
+    HEIGHT: 512
+    HFOV: 45
+    TYPE: HabitatSimRGBSensor
+  DEPTH_SENSOR:
+    WIDTH: 256
+    HEIGHT: 256
+TASK:
+  TYPE: VLN-v0
+  SUCCESS_DISTANCE: 2.0
+  SENSORS: ['INSTRUCTION_SENSOR']
```

Comment:
Worth to add explicitly supported actions:
` POSSIBLE_ACTIONS: ['MOVE_FORWARD', 'TURN_LEFT', 'TURN_RIGHT', 'STOP']`

Reply:
Does stop need to go first to keep stop on 0?

Reply:
Correct, edited my comment.